protocol/activity_center.go

```diff
@@ -48,14 +49,19 @@ func (n *ActivityCenterNotification) Valid() error {
 
 }
 
-func showMentionActivityCenterNotification(publicKey ecdsa.PublicKey, message *common.Message, chat *Chat, responseTo *common.Message) bool {
-	if chat != nil && !chat.Active {
-		return false
+func showMentionOrReplyActivityCenterNotification(publicKey ecdsa.PublicKey, message *common.Message, chat *Chat, responseTo *common.Message) int {
+	if chat == nil || !chat.Active || (!chat.CommunityChat() && !chat.PrivateGroupChat()) {
+		return -1
 	}
 
-	if message.Mentioned && chat != nil && (chat.CommunityChat() || chat.PrivateGroupChat()) {
-		return true
+	if message.Mentioned {
+		return ActivityCenterNotificationTypeMention
 	}
 
-	return false
+	publicKeyString := common.PubkeyToHex(&publicKey)
+	if responseTo != nil && responseTo.From == publicKeyString {
+		return ActivityCenterNotificationTypeReply
+	}
+
+	return -1
```

Comment:
In this case, probably you want to use `0`, and make it a constant, above in the  `iota` block, you can change it to:
```
const (	
ActivityCenterNotificationNoType = iota
ActivityCenterNotificationTypeNewOneToOne	
ActivityCenterNotificationTypeNewPrivateGroupChat
```

But probably at this point you could also return multiple values, like `(bool,AcitvityCenterType)` and return 0 if `bool` is `false`, probably more idiomatic, though not sure, maybe @Samyoul  has an opinion on the matter

Reply:
Updated. Thanks for the comment

Reply:
This is a fine approach.

Idiomatically enums are set with `iota + 1` because a nil int == `0`, but in this case `ActivityCenterNotificationNoType` is effectively the nil state, so the approach is good.